lib/errorMap.ts

```diff
@@ -1,6 +1,5 @@
-import * as zod from 'zod';
-
-import { fromZodIssue } from './fromZodIssue';
+import { fromZodIssue } from './fromZodIssue.ts';
+import type * as zod from 'zod';
```

Comment:
[note] Related to my other comment, see that the external dependency is second in order while the local dependency is first. This is because of the removal of eslint plugin. Personally, I'm fine either/or.

Reply:
This is wrong - not the intended behaviour I was after. Let me have a look...

Reply:
Ah, this is because it's a type import. The config it to keep type imports at the bottom of the list. This is expected.